### PR TITLE
Mitchell images database

### PR DESCRIPTION
**Pull request by bluesheepllama**
added images
added the blob image type to the back end

### CODE REVIEW

File: backend/src/entity/FoodItem.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { Blob } from "buffer"
```
Comment (kelseyp) [inline backend/src/entity/FoodItem.ts]: It doesn't look like the Blob import is being used. If not, this should be removed

File: backend/src/migration/initialize-database.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { parse } from 'csv-parse';
 import { AppDataSource } from "../data-source";
 import { FoodItem } from "../entity/FoodItem";
 import { Restaurant } from "../entity/Restaurant";
+import { Blob } from "buffer";
```
Comment (kelseyp) [inline backend/src/migration/initialize-database.ts]: Please remove unused imports.

File: backend/src/index.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@ AppDataSource.initialize().catch(error => console.log(error))
 const app = express()
 
 const corsOptions = {
-    origin: 'http://127.0.0.1:3000',
```
Comment (kelseyp) [inline backend/src/index.ts]: This was a local fix for your machine. Please revert to leave leave as-is in dev.

**Review by kelseyp — CHANGES_REQUESTED** [review]
Please review comments and complete changes before approval.

**Review by kelseyp — APPROVED** [review]

File: backend/src/migration/initialize-database.ts [hunk 4]
```diff
@@ -63,6 +64,7 @@ AppDataSource.initialize().then(async () => {
         foodItem.price = rawItem.Price;
         foodItem.description = rawItem.Description;
         foodItem.restaurant = restaurant;
+        foodItem.image = "../../../shared/data/images/" + rawItem.Image;
```
Comment (xulean) [inline backend/src/migration/initialize-database.ts:67]: it is a hard code; you can store the entire path in the record in the future.

**Review by xulean — APPROVED** [review]

**Pull request merged**